lib/grp.gi

```diff
@@ -31,7 +31,7 @@ InstallImmediateMethod( IsFinitelyGeneratedGroup,
 #M  IsCyclic( <G> ) . . . . . . . . . . . . . . . . test if a group is cyclic
 ##
 #  This used to be an immediate method. It was replaced by an ordinary
-#  method sine the flag is typically set when creating the group.
+#  method since the flag is typically set when creating the group.
```

Comment:
This fixes a typo from the first commit. It should be instead melded into the first commit...

Reply:
ditto